delta-lake/delta-spark350db143/src/main/scala/com/nvidia/spark/rapids/delta/shims/DeleteCommandMetaShim.scala

```diff
@@ -16,10 +16,28 @@
 
 package com.nvidia.spark.rapids.delta.shims
 
+import com.databricks.sql.transaction.tahoe.commands.DeletionVectorUtils
+import com.databricks.sql.transaction.tahoe.sources.DeltaSQLConf
 import com.nvidia.spark.rapids.delta.{DeleteCommandEdgeMeta, DeleteCommandMeta}
 
 object DeleteCommandMetaShim {
-  def tagForGpu(meta: DeleteCommandMeta): Unit = {}
+  def tagForGpu(meta: DeleteCommandMeta): Unit = {
+    val dvFeatureEnabled = DeletionVectorUtils.deletionVectorsWritable(
+      meta.deleteCmd.deltaLog.unsafeVolatileSnapshot)
+    if (dvFeatureEnabled && meta.deleteCmd.conf.getConf(
```

Comment:
I presume the part after `&&` is already implied by L25

Reply:
That's a global conf that can be set independently of the metadata from what I know